src/bytes.rs

```diff
@@ -59,7 +59,7 @@ where
 ///
 /// Return the read word and the remainder of the buffer
 ///
-/// # Panics
+/// # Safety
```

Comment:
This is not really a safety context. It is a similar situation to [copy_from_slice](https://doc.rust-lang.org/std/primitive.slice.html#panics-29)

I believe the safety topic should be used to address unsafe cases that will cause undefined behavior without panicking. One example is [as_chunks_unchecked](https://doc.rust-lang.org/std/primitive.slice.html#safety-3) where the safety topic gives precise instructions on how to properly constraint the function to avoid UB.